app/assets/stylesheets/_content_block.scss

```diff
@@ -2,4 +2,9 @@
   border-color: black;
   border-style: solid;
   border-width: 1px;
-}
\ No newline at end of file
+}
+
+//Fix for sticky footer
+div[role='main'] {
+  flex: 1 100%;
+}
```

Comment:
We should probably look at shifting this to a better home when we refactor, but as this is commented it should make it obvious where an appropriate home is.